### PR TITLE
Alar2 modifications

### PR DESCRIPTION
**Pull request by malachma**
This PR is required for PR "https://github.com/Azure/repair-script-library/pull/13" as the current vm-repair extension needs to be modified to add the build tools at deployment time with the help of a cloud-init script.

### CODE REVIEW

File: src/vm-repair/azext_vm_repair/scripts/linux-build_setup-cloud-init.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#cloud-config
+# cloud-init script to setup the build environment for alar2
```
Comment (qwordy) [inline src/vm-repair/azext_vm_repair/scripts/linux-build_setup-cloud-init.txt:2]:
```suggestion
# cloud-config
# cloud-init script to setup the build environment for alar2
```

**Review by qwordy — APPROVED** [review]

**Comment by qwordy** [comment]
@malachma What is Alar2

File: src/vm-repair/azext_vm_repair/scripts/linux-build_setup-cloud-init.txt (hunk 1 above)
Comment (malachma) [reply src/vm-repair/azext_vm_repair/scripts/linux-build_setup-cloud-init.txt:2]: The first line in the script for cloud-init has a special meaning. In this case we say that we want to use a cloud-config data  script. #could-config is not a comment instead it had to be treated as a statement. Like a macro in C.
The suggested code change does break the code therefore.

**Comment by qwordy** [comment]
Please fix the following errors
```
/home/vsts/work/1/s/src/vm-repair/azext_vm_repair/custom.py:69:36: E126 continuation line over-indented for hanging indent
/home/vsts/work/1/s/src/vm-repair/azext_vm_repair/custom.py:69:215: E202 whitespace before ')'
/home/vsts/work/1/s/src/vm-repair/azext_vm_repair/custom.py:72:36: E126 continuation line over-indented for hanging indent
/home/vsts/work/1/s/src/vm-repair/azext_vm_repair/repair_utils.py:26:1: E302 expected 2 blank lines, found 1
2     E126 continuation line over-indented for hanging indent
1     E202 whitespace before ')'
1     E302 expected 2 blank lines, found 1
```

File: src/vm-repair/azext_vm_repair/scripts/linux-build_setup-cloud-init.txt (hunk 1 above)
Comment (qwordy) [reply src/vm-repair/azext_vm_repair/scripts/linux-build_setup-cloud-init.txt:2]: Got it. Thank you!

**Comment by malachma** [comment]
We have 3 build errors which are reported to have failed. The log messages don't allow me to understand where exactly the issue is raised and why. Could this please be further  investigated.

**Comment by qwordy** [comment]
> 
> 
> We have 3 build errors which are reported to have failed. The log messages don't allow me to understand where exactly the issue is raised and why. Could this please be further investigated.

I can't see logs. It's gone. I reran them.

**Comment by malachma** [comment]
Thanks for running the build-test again.

**Pull request merged**